superset/assets/visualizations/nvd3_vis.css

```diff
@@ -35,3 +35,32 @@ text.nv-axislabel {
 text.nv-axislabel {
   font-size: 14px !important;
 }
+
+g.solid path, line.solid {
+  stroke-dasharray: unset;
+}
+
+g.dashed path, line.dashed {
+  stroke-dasharray: 5, 5;
+}
+
+g.wideDashed path, line.wideDashed {
```

Comment:
as mentioned above, I'd call this 'long dash', it doesn't affect the stroke width just the length of the dash.